e2e/tests/start_call.spec.ts

```diff
@@ -393,7 +393,7 @@ test.describe('switching products', () => {
         await expect(devPage.page.locator('#calls-widget')).toBeVisible();
 
         await devPage.page.locator('#calls-widget-participants-button').click();
-        const participantsList = devPage.page.locator('#calls-widget-participants-menu');
+        const participantsList = devPage.page.locator('#calls-widget-participants-list');
```

Comment:
Just curious, what did boards due to affect this?

Reply:
No idea to be honest I just wanted to fix it fast. Maybe something on `.Menu` class again. But it started failing all of the sudden, I guess it could have been webapp as well, didn't investigate further.